stories/documentation/overlays/popover/popover.stories.ts

```diff
@@ -63,25 +55,31 @@ const template: Story<PopoverStory> = (args: PopoverStory) => ({
 });
 
 export const basic = template.bind({});
-basic.args = {
-	popoverContent: 'popover content',
-	position: 'below',
-	alignment: 'center',
-	trigger: 'click',
-}
+basic.parameters = {
+	// Disable controls as they are not modifiable because of ComponentWrapper
+	controls: { include: [] },
+	docs: {
+		source: {
+			language: 'ts',
+			code: `
```

Comment:
It'd be nice to write this code block the same way (inline or via function calls) in all sb files